packages/components/src/utils/use-combined-refs.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useRef } from '@wordpress/element';
+
+export function useCombinedRefs( ...refs ) {
```

Comment:
should we use this instead of having our own https://www.npmjs.com/package/react-merge-refs

Reply:
I'm happy using that! Looks like @diegohaz helped with it 😎